code-challenges.test.js

```diff
@@ -15,42 +15,195 @@
+
 
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
 
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
-// Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
-// Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
+describe("fullHouseArray", () => {
+    it("takes in an array of 5 numbers and determines whether or not the array is a “full house", () => {
+        const hand1 = [5, 5, 5, 3, 3]
+        // Expected output: true
+        const hand2 = [5, 5, 3, 3, 4]
+        // Expected output: false
+        const hand3 = [5, 5, 5, 5, 4]
+        // Expected output: false
+        expect(fullHouseArray(hand1)).toEqual(true)
+        expect(fullHouseArray(hand2)).toEqual(false)
+        expect(fullHouseArray(hand3)).toEqual(false)
 
+    })
+})
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// ReferenceError: fullHouseArray is not defined
 
 // b) Create the function that makes the test pass.
+
+//Goal: Takes in an array of 5 numbers and determines whether or not the array is a “full house. 
+
+//Pseudocode:
+// Declare a function expression that takes an array as a parameter. (fullHouseArray).
+// Set the parameter as numbersArray.
+// Create for loop that defines "full house" values as true. 
+// Run the function using given parameters. 
+// Reutrn with the expected output. 
+
+const fullHouseArray= (numbersArray) => {
+    for (let i=0; i <numbersArray.length; i++) {
+        if (numbersArray[i] == [5, 5, 5, 3, 3]) {
+            return "true" 
+        } else {
+            return "false"
+        }
+    }
+}
+//* RAN TEST (RESULTS BELOW): *//
+
+// ● fullHouseArray › takes in an array of 5 numbers and determines whether or not the array is a “full house
+
+// expect(received).toEqual(expected) // deep equality
+
+// Expected: true
+// Received: "false"
+
+//   158 |         const hand3 = [5, 5, 5, 5, 4]
+//   159 |         // Expected output: false
+// > 160 |         expect(fullHouseArray(hand1)).toEqual(true)
+//       |                                       ^
+//   161 |         expect(fullHouseArray(hand2)).toEqual(false)
+//   162 |         expect(fullHouseArray(hand3)).toEqual(false)
+//   163 |
+
+
+// ** NOTES TO INSTRUCTOR **: Looks like I have the same issue going on here as I got for #1 - Part B, so for starters... I can assume there's a structure issue with my describe method - Part A for this one as well? 
```

Comment:
Hi there.  Actually, the test looks great!  There are a few problems with your function, the main one being that is not dynamic.  You don't actually want to check that the array === [5,5,5,3,3].  You want to check if there is a three of a kind and a pair.   I hope that helps.